automatic/lightalloy/update.ps1

```diff
@@ -0,0 +1,32 @@
+import-module au
+import-module .\..\..\extensions\chocolatey-fosshub.extension\extensions\Get-UrlFromFosshub.psm1

```

Comment:
I am not sure if this is necessary if its installed in the system but it wont harm either.